evm_body.yaml

```diff
@@ -379,6 +384,11 @@ eth_getProof:
                 description: array of storage-keys which should be proofed and included
                 items:
                   type: string
+                default:
+                  [
+                    '0x742d35Cc6634C0532925a3b844Bc454e4438f44e',
+                    '0x742d35Cc6634C0532925a3b844Bc454e4438f44f',
+                  ]
```

Comment:
will this work in the same request?

Reply:
Right now, it's not working but showing the right structure like this: 

https://alchemy-api-docs.readme.io/reference/eth-getproof-4

But that's not because the params are wrong but because we are showing the right structure.

Reply:
got it! thank you